### PR TITLE
Fix: translations not auto-populating in wait for response node

### PR DESCRIPTION
**Pull request by akanshaaa19**
target issue: https://github.com/glific/glific-frontend/issues/3113

### CODE REVIEW

**Comment by mdshamoon** [comment]
Tried creating a new node with interactive message got this error 
image

**Comment by akanshaaa19** [comment]
> Tried creating a new node with interactive message got this error image

you will have to switch to this branch in the BE: `adding-translations`

File: src/store/thunks.ts [hunk 1]
```diff
@@ -1101,7 +1101,27 @@ export const onUpdateRouter = (renderNode: RenderNode) => (
...
+    routerWithCases.cases.forEach((caseItem: any) => {
+      if (caseItem.translations) {
+        Object.entries(caseItem.translations).forEach(([language, translation]: any) => {
+          if (!updatedLocalization[language]) {
+            updatedLocalization[language] = {};
+          }
+
+          updatedLocalization[language][caseItem.uuid] = {
+            arguments: translation.arguments
+          };
```
Comment (mdshamoon) [inline src/store/thunks.ts:1117]: Can you explain this part. Not sure what's happening here.

**Review by mdshamoon — APPROVED** [review]
Functionality wise working as expected

File: src/store/thunks.ts (hunk 1 above)
Comment (akanshaaa19) [reply src/store/thunks.ts:1117]: this part is just updating the localization object with translations for the router

**Pull request merged**